config/jobs/image-pushing/k8s-staging-node-ipam-controller.yaml

```diff
@@ -0,0 +1,23 @@
+postsubmits:
+  kubernetes-sigs/node-ipam-controller:
+    - name: post-node-ipam-controller-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-network, sig-k8s-infra-gcb
+      decorate: true
+      branches:
+        - ^main$
+        - ^release-
+        # Build semver tags, too
+        - ^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
+      spec:
+        serviceAccountName: gcb-builder
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/image-builder:v20230711-e33377c2b4
+            command:
+              - /run.sh
+            args:
+              - --project=k8s-staging-node-ipam-controller
+              - --scratch-bucket=gs://k8s-staging-node-ipam-controller-gcb
```

Comment:
have we created this bucket already?

Reply:
see instructions https://github.com/kubernetes/k8s.io/blob/main/registry.k8s.io/README.md

ping infra people for guidance, @ameukam and @upodroid

Reply:
Thanks, send the PR https://github.com/kubernetes/k8s.io/pull/6645 for it.